modules/oe_content_sub_entity/modules/oe_content_sub_entity_author/tests/src/Kernel/AuthorFieldEntityTest.php

```diff
@@ -206,6 +206,16 @@ public function testAuthorsField(): void {
     $this->assertEquals('Org1', $links[4]->a->__toString());
     $this->assertEquals('Org2', $links[5]->a->__toString());
 
+    // Load all existing author entities and assert there are 4 entities.
+    $authors = $this->entityTypeManager->getStorage('oe_author')->loadMultiple();
+    $this->assertCount(4, $authors);
+    // Delete the organisation nodes referenced by $author4 entity.
+    $org_node1->delete();
+    $org_node2->delete();
```

Comment:
Both organisation nodes are being referenced by the `$authors4` entity. So the test asserts there are 4 author entity before removing the organisation nodes. After removing the organisation nodes there are only 3 author entity because the entity referencing them got deleted.

The person node is referenced by `$author3`, what is the relation?